### PR TITLE
✨E2E: increase timeout for video streaming and improve a bit logs about websockets

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

- increases the timeout when checking S4l videostreaming to 30 seconds
- logs warning if the websocket(s) are closed
- logs error if the websocket(s) have errors

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py [hunk 1]
```diff
@@ -263,28 +263,37 @@ def wait_for_pipeline_state(
...
+        ) as ctx:
+
+            def on_framesent(payload: str | bytes) -> None:
+                ctx.logger.debug("⬇️ %s", payload)
+
+            def on_framereceived(payload: str | bytes) -> None:
+                ctx.logger.debug("⬆️ %s", payload)
+
+            def on_close(payload: WebSocket) -> None:
+                ctx.logger.warning("Websocket closed: %s", payload)
```
Comment (pcrespov) [inline packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py:282]: no emoticos for close or error? :-(

- **Closed**: 🚪 (Door)
- **Error**: ❌ (Cross Mark)

**Review by pcrespov — APPROVED** [review]
thx 🙏

**Review by odeimaiz — APPROVED** [review]
Merci

File: packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py (hunk 1 above)
Comment (sanderegg) [reply packages/pytest-simcore/src/pytest_simcore/helpers/playwright.py:282]: where do you get those?

**Review by matusdrobuliak66 — APPROVED** [review]
thanks!

**Review by GitHK — APPROVED** [review]

**Pull request merged**